src/atoms/pill/theme.ts

```diff
@@ -0,0 +1,16 @@
+import { ThemeType } from '../../util';
+import { IBoxTheme, ITextTheme } from '../../particles';
+
+export interface IPillThemeBase extends ThemeType {
+  text: ITextTheme;
+  background: IBoxTheme;
+}
+
+export interface IPillThemeState extends ThemeType {
+  default: IPillThemeBase;
+  // NOTE(rikhil): we can include hover, press, focus if we allow pills to be clicked
```

Comment:
Do you want these to be clickable?

Reply:
dont think so, they just contain information? I think if someone wants a clickable one they should use button. What do you think?

Reply:
Comment inspired by what they have here https://material-ui.com/components/chips/.